app/src/main/res/layout/call_item.xml

```diff
@@ -23,53 +25,38 @@
     android:id="@+id/relative_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:layout_weight="1"
-    android:orientation="vertical">
+    android:orientation="vertical"
+    android:gravity="center">
+
+    <com.facebook.drawee.view.SimpleDraweeView
+        android:id="@+id/avatarImageView"
+        android:layout_width="80dp"
+        android:layout_height="80dp"
+        android:layout_centerInParent="true"
+        app:roundAsCircle="true" />
 
     <TextView
         android:id="@+id/peer_nick_text_view"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignParentStart="true"
-        android:layout_alignParentTop="true"
-        android:layout_marginStart="8dp"
-        android:layout_marginEnd="8dp"
+        android:layout_alignParentBottom="true"
         android:layout_marginBottom="8dp"
-        android:textColor="@android:color/white"
-        android:layout_centerHorizontal="true"/>
-
-    <ImageView
-        android:id="@+id/remote_audio_off"
-        android:layout_width="16dp"
-        android:layout_height="16dp"
-        android:layout_below="@id/peer_nick_text_view"
-        android:layout_alignParentStart="true"
         android:layout_marginStart="8dp"
-        android:layout_marginEnd="8dp"
-        android:src="@drawable/ic_mic_off_white_24px"
+        android:textColor="@android:color/white" />
+
+    <org.webrtc.SurfaceViewRenderer
+        android:id="@+id/surface_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
         android:visibility="invisible" />
 
     <ImageView
-        android:id="@+id/remote_video_off"
+        android:id="@+id/remote_audio_off"
```

Comment:
you removed the `remote_video_off` option to disable a remote persons video and the audio thing replaces the video one?

Reply:
the `remote_video_off` icon was shown whenever a participant disabled it's own video. so at the same moment his video is replaced with his avatar. So the `remote_video_off` icon made no sense at all?

Reply:
Right, I see it now. So okay for now and a feature to add later.